crates/edition/src/lib.rs

```diff
@@ -1,6 +1,5 @@
 //! The edition of the Rust language used in a crate.
-// Ideally this would be defined in the span crate, but the dependency chain is all over the place
-// wrt to span, parser and syntax.
+// This should live in a separate crate because we use it in both actual code and codegen.
```

Comment:
Can we just `include!` this file in xtask instead? I personally don't really like this small thing being a crate. Or just duplicate what we need given its not much

Reply:
I don't want to duplicate because then adding edition will need to touch both. But `include!()` seems fine.

Reply:
It works for rustc but r-a is unable to find the file both with `#[path = "..."] mod edition;` and with `include!()`, do you know why?

Reply:
Ahaha, right. This fails in r-a because the include lies outside of the source root of the crate itself so we fail to resolve the path... This is one of the outstanding problems with our VFS system

Reply:
Done.

@rustbot ready